src/main/java/gift/model/Member.java

```diff
@@ -28,23 +30,35 @@ public Member(Member member, String activeToken) {
         this.email = member.email;
         this.password = member.password;
         this.activeToken = activeToken;
+        this.points = member.points;
     }
 
     public Member(Member member) {
         this.id = member.id;
         this.email = member.email;
         this.password = member.password;
+        this.points = member.points;
     }
 
     public Member(String email, String password) {
         this.email = email;
         this.password = password;
+        this.points = 0;
     }
 
     public Member(String email, String password, String activeToken) {
         this.email = email;
         this.password = password;
         this.activeToken = activeToken;
+        this.points = 0;
+    }
+
+    public Member(Long id, String email, String password, String activeToken, int points) {
+        this.id = id;
+        this.email = email;
+        this.password = password;
+        this.activeToken = activeToken;
+        this.points = points;
     }
```

Comment:
생성자가 많아진다면 아래와 같은 방식으로도 관리해볼 수 있을 것 같아요 :  
```suggestion
    public Member(String email, String password, String activeToken) {
        this(
            null,
            email,
            password,
            activeToken,
            0
        );
    }

    public Member(Long id, String email, String password, String activeToken, int points) {
        this.id = id;
        this.email = email;
        this.password = password;
        this.activeToken = activeToken;
        this.points = points;
    }
```